File: Interview Question (easy)/Interview Question (easy).js
function getStrings(city){
  city = city.toLowerCase().replaceAll(" ","")
  let count = []
  city.split("").map(val => count[val] = (count[val] || 0)  + 1);
  let result = []
  for(let [key,value] of Object.entries(count)){
    result.push(`${key}:${"*".repeat(value)}`)
  }
  return result.join(",")
}

File: Interview Question (easy)/Interview Question (easy).py
from collections import Counter
def get_strings(city):
    city = city.lower().replace(" ","")
    freq = Counter(city)
    result = []
    for s in city:
        tmp = s.lower()+":"+freq[s.lower()]*"*"
        if tmp not in result:
            result.append(tmp)
    return ",".join(result)
